packages/smooth_app/lib/database/product_query.dart

```diff
@@ -42,12 +45,22 @@ abstract class ProductQuery {
 
   /// Device Id, potentially used as API uuid.
   static String? deviceId;
```

Comment:
Please don't be afraid to remove `deviceId`. It's not a device id, and anyway it's the same as `OFFConfig.uuid`.